fiftyone/utils/video.py

```diff
@@ -519,6 +519,26 @@ def sample_frames_uniform(
     return sample_frames
 
 
+def concat_videos(input_paths, output_path, verbose=False):
```

Comment:
I think `concat` is a bit more clear here. "Merge" typically means some non-trivial combination of things.